pipeline/runtime/step_executor.go

```diff
@@ -307,6 +307,9 @@ func (e *StepExecutor) run(ctx context.Context, engine *engine.Engine, r *api.St
 	if r.Kind == api.Run {
 		return executeRunStep(ctx, engine, r, out, tiConfig)
 	}
+	if r.Kind == api.RunTestsV2 {
+		return executeRunTestsV2Step(ctx, engine, r, out, tiConfig)
+	}
```

Comment:
Will RunTestV1 work here, I see there is no condition for that?

Reply:
Yeah if it is neither then the final return is executeRunTestStep. I have tested it and it works. This is the same as on lite-engine

Reply:
I think we do have condition for V1 in LE 
<img width="981" alt="Screenshot 2024-08-01 at 10 59 28 PM" src="https://github.com/user-attachments/assets/155559d7-7b7d-43d7-8f60-96b235baac4a">

Reply:
Yes it is the same here